### ISSUE TITLE
cross platform testing

### ISSUE THREAD

**Issue by jacklein** [jacklein/rn-bottom-drawer#1]
My laptop is too old to download xcode so I cannot test on an iphone x simulator to see what it looks like. I'm also working on setting up the android simulator but if anyone could test on these devices and see if there's any problems that would be great!

**Comment by SoftARV** [comment]
i was testing the component on my Android phone and the gestures doesn't seen to work. 
I have to manually change the startUp property to see my component. 
Is there something else i need to do to get it working?
<img src="https://user-images.githubusercontent.com/12791948/50315896-82e7ce00-0482-11e9-94a7-b0bba2a15488.png" alt="drawing" width="200"/><img src="https://user-images.githubusercontent.com/12791948/50315916-8ed39000-0482-11e9-9dcd-80b7b398d4dc.png" alt="drawing" width="200"/>

**Comment by jacklein** [comment]
@SoftARV can you post your code so i can look at it? i can't be sure without looking at what you've written

**Comment by SoftARV** [comment]
this is the render method of my component.

The constant for the sizes are 400 for the container height and 60 for the bar.

![screen shot 2018-12-20 at 8 48 07 pm](https://user-images.githubusercontent.com/12791948/50320045-92721180-0498-11e9-82c0-fa1fdf94b2a3.png)

**Comment by jacklein** [comment]
downDisplay should just be {MAIN_BAR_SIZE}

tell me if this works, and if so, did the documentation confuse you? how could we change it to make it more clear?

**Comment by jacklein** [comment]
is your MAIN_BAR at the bottom? can you try setting offset={MAIN_BAR_SIZE}, and removing downDisplay. if you have a header or tab bar, you need to set the offset. let me know if that works.

**Comment by jacklein** [comment]
because you have a MAIN_BAR, you need to also set the offset={MAIN_BAR_SIZE}, let me know if that works.

**Comment by SoftARV** [comment]
I didn't set the offset because I'm not using any navbar or tabbar in my home screen as you can see in the images a put before.
I'm using the whole screen for the drawer but when a try to drag my component nothing happen.
maybe my component that is rendering in the drawer is pushing the draggable part away, I think?

**Comment by jacklein** [comment]
is there any way for you to post your entire code base or at least enough for me to be able to reproduce your problem?

**Comment by SoftARV** [comment]
let me upload my project to GitHub.
I was planning on doing it but never have the necessity yet.

**Comment by SoftARV** [comment]
It only has the height of the container at 400.
Here is the repo of my project 
https://github.com/SoftARV/Todo-App

**Comment by jacklein** [comment]
i bet it has something to do with the bottom drawer's parent component. try taking it out of the view and instead make it it's sibling?

**Comment by jacklein** [comment]
actually sorry i cant play with your project because i have to use the Expo ide which can only run javascript. if you can play around with it more and try to debug it i would be very grateful.

**Comment by SoftARV** [comment]
sure no problem, I figure it out.
if anything change I'll let you know.
and yeah I'm not using expo right now, just learning all the ropes with React Native only.

**Comment by jacklein** [comment]
yeah unfortunately my mac os is too old for xcode so theres not much i can do about it haha. 

and ok, keep me updated! and if you think it's a bug within this repo, feel free to make a PR!!

**Comment by zacler** [comment]
@jacklein - we are having some issues using this - animations are off and it doesn't scroll - any thoughts?

**Comment by jacklein** [comment]
@zacler sorry I’ve been busy starting a new job and I’ve been quite unresponsive :/ but have you looked at the solution posed on the other open issue?

**Comment by zacler** [comment]
It’s okay - congrats on the new job.

We went with another build I believe but will check.

Best,

Zach
